ci/teamcity/comment_on_pr.py

```diff
@@ -16,32 +16,67 @@
     pr_id = int(os.environ["GITHUB_PR_NUMBER"].split("/")[-1])
 except Exception:
     sys.exit(0)
-header = """
-<h2 align="center">:robot: TeamCity test results bot :robot:</h2>
 
-"""
+engine = os.environ["MODIN_ENGINE"]
 
-pytest_outputs = ["ray_tests.log", "dask_tests.log", "python_tests.log"]
+header = """<h1 align="center"><img width=7% alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/86/Teamcity_Logo.png/600px-Teamcity_Logo.png">
+    TeamCity {} test results bot</h1>\n\n""".format(
+    engine.title()
+)
+if engine == "ray":
+    pytest_outputs = ["ray_tests.log"]
+elif engine == "dask":
+    pytest_outputs = ["dask_tests.log"]
+elif engine == "python":
+    pytest_outputs = ["python_tests.log"]
+else:
+    raise Exception("Unknown Engine, set `MODIN_ENGINE` environment variable")
 
-full_comment = header
+full_comment = ""
+# Do not include coverage info in PR comment
+split_by_first = (
+    "----------- coverage: platform linux, python 3.7.5-final-0 -----------"
+)
+split_by_second = "--------------------------------------------------------------------------------------"
 
 for out in pytest_outputs:
-
-    full_comment += "<details><summary>{} Tests</summary>\n".format(
-        out.split("_")[0].title()
-    )
+    full_comment += "<details><summary>Tests Logs</summary>\n"
     full_comment += "\n\n```\n"
-    full_comment += open(out, "r").read()
+    content = open(out, "r").read()
+    full_comment += "".join(
+        "".join(
+            [
+                i.split(split_by_first)[0],
+                i.split(split_by_first)[-1].split(split_by_second)[-1],
+            ]
+        )
+        for i in content.split("+ python3 -m pytest -n=48 ")
```

Comment:
Is it possible not to hardcode `-n=48` here? It may make it impossible to change number of workers which we had to do just recently because of ray race condition bug. Also servers with different number of CPUs may require different number of workers.